clisops/core/subset.py

```diff
@@ -791,7 +786,7 @@ def shape_bbox_indexer(ds, poly):
     # Find indices nearest the rectangle' corners
     # Note that the nearest indices might be inside the shape, so we'll need to add a *halo* around those indices.
     if rectilinear:
-        if version.parse(xarray.__version__) < version.Version("2022.06.0"):
+        if version.parse(xarray.__version__) < version.Version("2022.6.0"):
```

Comment:
The fix for xarray can be found here

Reply:
I think you'll need to rename `.indexers` to `.dim_indexers` below too!